test/utils/url.test.js

```diff
@@ -0,0 +1,72 @@
+import { appendQuery } from 'src/utils';
+import nock from 'nock';
+import expect from 'expect';
+
+/**
+ * @private
+ * Adapted from https://github.com/lakenen/node-append-query
+ */
+describe('appendQuery', function(){
+    it('should append querystring to queryless url', function () {    
+        var result = appendQuery('http://example.com/foo', 'bar=baz&beep=boop')
+            , expected = 'http://example.com/foo?bar=baz&beep=boop'
```

Comment:
Re: the style doc - I've tried to keep the tests as similar to the original as possible, so we can compare later if required. 

Re: es6 code, again a good question for @thomasconner !